### ISSUE TITLE
Client settings sync or export/import

### ISSUE THREAD

**Issue by nextMJ** [owncloud/client#10991]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Feature description

I just moved to a new computer and when installing the client, I had to retype all the client settings which I had on the old PC to new PC (mainly a long list of ignored files). It would be very beneficial if it is possible to easily export these settings (json?) and import them to the new app.

OR it can be extended as many other software already does - the settings could be synced on the server and could be easily downloaded after the first login.

### Proposed solution

Already above.

**Comment by hodyroff** [comment]
The ignore settings are in one file, why not just copy that one file? Most people don't do any changes to that one, to be honest ... so its a little bit of a special thing. So did you consider to just copy that file?

**Comment by nextMJ** [comment]
Well, I did not know that this setting is in some file at some location which is easy to transfer. The app does not provide any help in this way. So no, I did not consider this option even if it is probable very easy.

If most people do not make changes in that list, the "edit" button has the same level of importance as import/export. OR as proposed - this list could be synced to server user account which would be the best.

**Comment by TheOneRing** [comment]
The settings mostly consists of the server settings and the local directories.
Then a hand full of user settings.
- We can't sync the server settings to the server.
- We cant export the list of local directories as the new system might have another setup (disks, drives, os...)

So your question was mainly about exporting/syncing the ignore list?

**Comment by nextMJ** [comment]
I agree that exporting setup about local directories is difficult due to different system setup and I do not have idea about the solution here.

My original idea was with the ignore list but I had to change some other stuff that relates to the client too (but they were not that painful). I think that most of the stuff in the client "settings" tab is related to the client and user usually (I think) transfers everything when having a new computer since the world setup (country, network, ..) did not change (language, proxy, virtual files settings, handling large files, network speed limits).  But those other settings were easy to set so I did not pay attention too much to them. But exporting/importing those would be beneficial for me as well. So from my programmer's point of view - having the option to export the JSON file with the setup should be possible with one click and rewriting the setup based on the imported file should be also possible (maybe also during installation?). In addition, syncing this file to the server should be also possible but I agree that it would be probably a more difficult task to solve (different clients setup for a single user - PC, phone, etc.). 

But it was just an idea. I hope that with my new computer, I will have to solve this trouble in years from now. I just thought that it could be beneficial for all other users. Especially for admins in companies (single shared settings at the install time). But I'm not the one to decide if this is useful or not :-)
